tests/main.nf.test

```diff
@@ -564,5 +564,64 @@ nextflow_workflow {
             }
         },
 
+        test("cycle: no illumination correction, backsub") {
+
+            when {
+                params {
+                    segmentation = "mesmer"
+                    backsub = true
+                }
+                workflow {
+                    """
+                    input[0] = Channel.of(
+                        [
+                            [id:"TEST1", cycle_number:1, channel_count:4],
+                            "https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/imaging/background_subtraction/cycif_tonsil_registered.ome.tif",
+                            [],
+                            [],
+                        ],
+                        [
+                            [id:"TEST2", cycle_number:2, channel_count:4],
+                            "https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/imaging/background_subtraction/cycif_tonsil_registered.ome.tif",
+                            [],
+                            [],
+                        ],
+                    )
+                    input[1] = Channel.of(
+                        [
+                            [1,1,'DNA 1',[],[],[],100,[],[]],
+                            [2,1,'Na/K ATPase',[],[],[],100,[],[]],
+                            [3,1,'CD3',[],[],[],100,[],[]],
+                            [4,1,'CD45RO',[],[],[],100,[],[]],
+                            [5,2,'DNA 2',[],[],[],100,[],'TRUE'],
+                            [6,2,'Antigen Ki67',[],[],[],100,'Na/K ATPase',[]],
+                            [7,2,'Pan-cytokeratin',[],[],[],100,'CD3',[]],
+                            [8,2,'Aortic smooth muscle actin',[],[],[],100,'CD45RO',[]]
+                        ],
+                    )
+                    """
+                }
+            }
+
+            then {
+                assertAll (
+                    {
+                        assert snapshot (
+                            path("$outputDir/registration/ashlar/TEST1.ome.tif"),
+                            path("$outputDir/registration/ashlar/TEST2.ome.tif"),
+                            path("$outputDir/segmentation/deepcell_mesmer/mask_TEST1.tif"),
+                            path("$outputDir/segmentation/deepcell_mesmer/mask_TEST2.tif"),
+                            CsvUtils.roundAndHashCsv("$outputDir/quantification/mcquant/mesmer/TEST1_mask_TEST1.csv"),
```

Comment:
Might as well go straight to `summarizeCsv` for these two, since it's looking like they already fail to match on my system (irrespective of the id:TEST2 problem above).

Reply:
sounds good. I made the change.